tests/core/pm-module/test_pm_init.py

```diff
@@ -37,32 +38,73 @@ def web3():
 
 
 @pytest.fixture
-def owned_manifest():
-    with open(str(V2_PACKAGES_DIR / 'owned' / '1.0.0.json')) as file_obj:
-        return json.load(file_obj)
-
-
-@pytest.fixture
-def standard_token_manifest():
-    with open(str(V2_PACKAGES_DIR / 'standard-token' / '1.0.0.json')) as file_obj:
-        return json.load(file_obj)
+def get_ethpm_manifest():
+    def _get_ethpm_manifest(name, filename):
+        return get_manifest(name, filename)
+    return _get_ethpm_manifest
```

Comment:
You shouldn't need the inner function and should be able to just return the `get_manifest` function.

Reply:
This is the pattern i've been using when I want to pass arguments into a fixture. I tried 
```python
def get_ethpm_manifest(name, filename):
    return get_manifest(name, filename)
```
But that doesn't fly since it expects `name` and `filename` to be available as fixtures. Is that what you were suggesting? Or something different?

Reply:
I'm suggesting you write it as:

```python
@pytest.fixture
def get_ethpm_manifest():
    return get_manifest
```

Though this is really just exposing a function that *can* be imported as a fixture instead.  I don't necessarily think this is a good pattern as you could instead, in your testing module just write:

```python
from ethpm.assets import get_manifest as get_ethpm_manifest

...
```

Doesn't seem like the fixture adds a lot of value....